financialaid/api.py

```diff
@@ -45,6 +47,27 @@ def determine_auto_approval(financial_aid):
     return financial_aid.income_usd > income_threshold or income_threshold == 0
 
 
+def determine_income_usd(original_income, original_currency):
+    """
+    Take original income and original currency and converts income from the original currency
+    to USD.
+    Args:
+        original_income: original income, in original currency (for a FinancialAid object)
+        original_currency: original currency, a three-letter code
+    Returns:
+        income_usd: the original income converted to US dollars
```

Comment:
The format here should be `float: the original income...` where the word before the `:` is a type